packages/application-shell/src/components/fetch-project/fetch-project.spec.js

```diff
@@ -20,132 +39,19 @@ const renderProject = (options) =>
     options
   );
 
-const createGraphqlResponseForProjectQuery = (custom) => ({
-  loading: false,
-  project: {
-    __typename: 'Project',
-    key: 'test-1',
-    version: 1,
-    name: 'Test 1',
-    countries: ['de'],
-    currencies: ['EUR'],
-    languages: ['de'],
-    initialized: true,
-    expiry: {
-      __typename: 'ProjectExpiry',
-      isActive: true,
-      daysLeft: null,
-    },
-    suspension: {
-      __typename: 'ProjectSuspension',
-      isActive: false,
-      reason: null,
-    },
-    allAppliedPermissions: [
-      {
-        __typename: 'AppliedPermission',
-        name: 'canManageProjectSettings',
-        value: true,
-      },
-    ],
-    allAppliedDataFences: [
-      {
-        __typename: 'StoreDataFence',
-        value: 'usa',
-        group: 'orders',
-        name: 'canManageOrders',
-        type: 'store',
-      },
-    ],
-    allAppliedActionRights: [
-      {
-        __typename: 'AppliedActionRight',
-        group: 'products',
-        name: 'canEditPrices',
-        value: true,
-      },
-    ],
-    allAppliedMenuVisibilities: [
-      {
-        __typename: 'AppliedMenuVisibilities',
-        name: 'hideDashboard',
-        value: false,
-      },
-    ],
-    owner: {
-      __typename: 'Organization',
-      id: 'owner-id',
-      name: 'commercetools',
-      createdAt: '2019-01-01T00:00:00.000Z',
-    },
-    settings: {
-      __typename: 'ProjectSetting',
-      id: 'settings-id',
-      productSettings: ['product-settings-id-1'],
-      currentProductSettings: 'product-settings-id-1',
-    },
-  },
-  ...custom,
-});
-
 describe('rendering', () => {
   it('should fetch project and pass data to children function', async () => {
-    const rendered = renderProject({
-      mocks: [
-        {
-          request: {
-            query: ProjectQuery,
-            variables: {
-              target: GRAPHQL_TARGETS.MERCHANT_CENTER_BACKEND,
-              projectKey: 'test-1',
-            },
-          },
-          result: {
-            data: createGraphqlResponseForProjectQuery(),
-          },
-        },
-      ],
-    });
-    await rendered.findByText(/Test 1/i);
-  });
-  it('should render loading state', async () => {
```

Comment:
I'm not sure how we can test the loading state now. The question also is if we actually need to or just skip that.

Reply:
Nvm, I figured that out